wrangler-core/src/test/java/io/cdap/wrangler/TestingPipelineContext.java

```diff
@@ -35,46 +35,19 @@
  * {@link Executor} execution.
  */
 public class TestingPipelineContext implements ExecutorContext {
-  private final StageMetrics metrics;
-  private final String name;
-  private final TransientStore store;
-  private final Map<String, String> properties;
+  private final StageMetrics metrics = new MockStageMetrics();
+  private final String name = "testing";
+  private final TransientStore store = new DefaultTransientStore();
+  private final Map<String, String> properties = new HashMap<>();
+  private final boolean schemaManagementEnabled;
 
   public TestingPipelineContext() {
-    name = "testing";
-    properties = new HashMap<>();
-    store = new DefaultTransientStore();
-    metrics = new StageMetrics() {
-      @Override
-      public void count(String s, int i) {
-
-      }
-
-      @Override
-      public void gauge(String s, long l) {
-
-      }
-
-      @Override
-      public void pipelineCount(String s, int i) {
-
-      }
-
-      @Override
-      public void pipelineGauge(String s, long l) {
-
-      }
-
-      @Override
-      public Metrics child(Map<String, String> tags) {
-        return this;
-      }
+    this.schemaManagementEnabled = false;
+  }
 
-      @Override
-      public Map<String, String> getTags() {
-        return Collections.emptyMap();
-      }
-    };
+  public TestingPipelineContext(boolean schemaManagementEnabled) {
```

Comment:
nit: consider using setter. `new TestingPipelineContext().setSchemaManagementEnabled(true)` is more clear when reading code. Boolean values are generally not very readable